spec/mailers/ticket_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe TicketMailer, type: :mailer do
+  let(:user) { create(:user) }
+  let(:ticket) { create(:ticket, user:) }
+
+  it 'replies with an error message email' do
```

Comment:
`replies with an error message email` ? why? I'm mean there is no _context_ explaining why this would produce an error and I can't deduce it from the code, neither. Only see a check on the deliveries. Maybe missing some
`context 'when ....' do` ?

Reply:
done